providers/implementations/encode_decode/encode_key2any.c

```diff
@@ -489,6 +572,11 @@ static int prepare_ec_explicit_params(const void *eckey,
     return 1;
 }
 
+/*
+ * This implements EcpkParameters, where the CHOICE is based on whether there
+ * is a curve name (curve nid) to be found or not.  See RFC 3279 for details.
+ * TODO: shouldn't we use i2d_ECPKParameters()?
```

Comment:
Is this going to resolved?

Reply:
I have no idea.  I didn't put together these functions.